examples/legacy/vsphere-kubeadm.yaml

```diff
@@ -0,0 +1,1490 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
```

Comment:
Again user will see WARNING that Cluster resource has reference to non-existing VSphereCluster